### PR TITLE
create-exercise can generate README.md

### LINKED ISSUE #62: Update create-exercise to create README.md if description.md exists
If the specification in exercism/problem-specifications has a description.md, that can be used to make a README for the generated exercise.

Additionally, from the metadata.yml file, we can take the values for source and source_url and add them to the end of the README.md file.

The easiest way would be to [use the configlet tool within create-exercise to generate the README](https://github.com/exercism/docs/blob/master/language-tracks/exercises/anatomy/readmes.md#generation) if the problem-specification repo was referenced

### PR DESCRIPTION
**Pull request by fwip**
This closes #62.

Not a lot of exciting things going on in this pull request. Generates `README.md` using `configlet generate`, and factors out a pattern for running processes.

### CODE REVIEW

**Comment by Stargator** [comment]
I think this would be a good PR to add those updates you removed from the other one. Like setting the test dependency to "<0.13.0" and I think there may have been other small fixes.

**Comment by fwip** [comment]
The test version already got updated in #51. The only other change that I can recall was changing the vim modeline to set filetype instead of syntax, but I don't know if that's a good change to make.

**Comment by Stargator** [comment]
@fwip Understood.

File: tool/create-exercise [hunk 1]
```diff
@@ -226,38 +242,32 @@ Future main(args) async {
...
+      stdout.write("Successfully created README.md\n");
+    } else {
+      stderr.write("Warning: `configlet generate` exited with an error, 'README.md' is likely malformed.\n");
     }
 
-    if (res.exitCode != 0) {
-      stderr.write("Warning: dartfmt exited with a code of ${res.exitCode}, '$testFileName' is likely malformed.\n");
-    } else {
+    // The output from test generation is not always well-formatted, use dartfmt to clean it up
+    final fmtSuccess = runProcess("dartfmt", ["-l", "120", "-w", testFileName]);
```
Comment (Stargator) [inline tool/create-exercise]: Recommend not using testFileName, but the path to the new exercise ie : "$dartRoot/exercises/pangram"

Comment (fwip) [reply tool/create-exercise]: Makes sense!

**Comment by fwip** [comment]
Okay, I've fixed that error message (was forgetting to `await` for async code) and changed dartfmt to format all generated dart files.

**Comment by Stargator** [comment]
Okay, it's been 9 days. I'm going to merge this in about 4 hours, unless someone suggests additional changes. @fwip Thanks so much for all your work on this tool.

**Pull request merged**